src/repositories/auth/repository.py

```diff
@@ -36,6 +48,20 @@ def verify_bot_token(cls, auth_token: str) -> VerificationResult:
         else:
             return VerificationResult(success=False)
 
+    @classmethod
+    async def verify_user_token(cls, token: str) -> VerificationResult:
+        try:
+            payload = cls.decode_token(token)
+            innohassle_id: str = payload.get("uid")
+            if innohassle_id is None:
+                return VerificationResult(success=False)
+            user_data = await innohassle_accounts.get_user_by_innohassle_id(innohassle_id)
+            if user_data is None:
+                raise VerificationResult(success=False)
+            return VerificationResult(success=True, telegram_id=user_data.telegram.id)
```

Comment:
If user_data.telegram = None it will fail and raise 500 exception. The are two ways in such scenario: return `VerificationResult(success=False)` or set `VerificationResult(success=True, telegram_id=None)`

Reply:
I guess, telegram_id=None will mess up logic.